model-optimizer/extensions/middle/ApplyPermutations.py

```diff
@@ -126,8 +129,6 @@ def permute_op_nodes_attrs(graph: Graph):
 
     @staticmethod
     def permute_input_data(graph: Graph):
-        if graph.graph['layout'] != 'NHWC':
```

Comment:
Did this check was removed to allow weights permutation for Convolutions for models with NCHW layout?

Reply:
We do not only change layout of data paths with this method, but any path, including weights.